File: src/ssr_pkg/src/yao_node.cpp
#include<ros/ros.h>
#include<std_msgs/String.h>
int main(int argc,char  *argv[])
{
    ros::init(argc,argv,"yao");
    printf( "过去生。。。。\n");
    ros::NodeHandle nh;
    ros::Publisher pub=nh.advertise<std_msgs::String>("qiu_dai_fei",10);
    ros::Rate loop_rate(10);
    while (ros::ok())
    {
        printf("yao_刷屏\n");
        std_msgs::String msg;
        msg.data="求带飞\n";
        pub.publish(msg);
        loop_rate.sleep();
    }
    
    return 0;
}

File: src/ssr_pkg/scripts/yao_node.py
#!/usr/bin/env python3 
#coding=utf-8
 
import rospy
from std_msgs.msg import String

if __name__=="__main__":
    rospy.init_node("yao_node")
    rospy.logwarn("yao_node is started ")

    pub=rospy.Publisher("yao_node_toptic",String,queue_size=10)

    rate=rospy.Rate(1)

    while not rospy.is_shutdown():
        rospy.loginfo("yao_node is running")
        
        msg=String()
        msg.data="This is yao_node publishing data"
        
        pub.publish(msg)
        rate.sleep()